libcodechecker/server/config_database.py

```diff
@@ -4,41 +4,37 @@
 #   License. See LICENSE.TXT for details.
 # -------------------------------------------------------------------------
 """
-Database server handling.
+Database server handling for the server's configuration database.
```

Comment:
But we could pass the schema as an argument, it could always raise an exception and in case the exception is from the product database, the caller could bring down the server. And we have utilities to convert between connection strings and command line arguments. In this case, my gut feeling is that the need to maintain the code twice is worse than abstracting these away.